debezium-core/src/main/java/io/debezium/data/Enum.java

```diff
@@ -24,22 +26,51 @@
      * 
      * @param allowedValues the comma separated list of allowed values; may not be null
      * @return the schema builder
+     * @deprecated Use {@link #builder(List)}
      */
+    @Deprecated
     public static SchemaBuilder builder(String allowedValues) {
         return SchemaBuilder.string()
                             .name(LOGICAL_NAME)
                             .parameter(VALUES_FIELD, allowedValues)
                             .version(1);
     }
 
+    /**
+     * Returns a {@link SchemaBuilder} for an enumeration. You can use the resulting SchemaBuilder
+     * to set additional schema settings such as required/optional, default value, and documentation.
+     *
+     * @param allowedValues list of allowed values; may not be null
+     * @return the schema builder
+     */
+    public static SchemaBuilder builder(List<String> allowedValues) {
+        return SchemaBuilder.string()
+                            .name(LOGICAL_NAME)
+                            .parameter(VALUES_FIELD, "[" + String.join(",", allowedValues) + "]")
```

Comment:
Does this alter the public representation of the enum values (brackets added)? If so, why is it? Strictly speaking, it's a breaking change, so if possible, we should not do it.

To give some background: while DBZ doesn't have much of an actual API/SPI in terms of Java interfaces (the embedded engine being the big exception), the message structure we produce is our public contract and we go to great length to evolve it in a compatible fashion. This applies to the payload as well as the schema of messages.

Reply:
Since I decided against `Enum#builder(List)`, this is no longer a concern and the stored string should be considered backward compatible.

Previously if the `ENUM` options were say the following:
`'station', 'post_office', 'a,b', 'c,\'d'`

We would have originally parsed and stored in `allowed` as `station,post_office`.  You'll notice that the last two options since they contained a `,` (comma) they were excluded.  With the code changes this provides, this will now be stored in `allowed` as `station,post_office,a\,b,c\,'d`.